### PR TITLE
Add keys only seek for badger

### PR DESCRIPTION
**Pull request by lazynina**

### CODE REVIEW

**Comment by lazynina** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1135?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#1135** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1135?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#1134** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1134?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1127** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1127?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#1128](https://github.com/deso-protocol/core/pull/1128) <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1128?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#1116** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1116?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1115** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1115?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1110** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1110?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#1113](https://github.com/deso-protocol/core/pull/1113) <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1113?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#1109** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1109?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `feature/pos-networking-and-syncing`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @lazynina and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lib/db_utils.go [hunk 1]
```diff
@@ -11915,6 +11916,95 @@ func _enumerateKeysForPrefixWithLimitOffsetOrderWithTxn(
 	return keysFound, valsFound, nil
 }
 
+func EnumerateKeysOnlyForPrefixWithLimitOffsetOrderAndSkipFunc(
```
Comment (tholonious) [inline lib/db_utils.go:11919]: Do you just want to replace the old `EnumerateKeysForPrefixWithLimitOffsetOrderAndSkipFunc` with this function? The old one is dead code now

**Review by tholonious — APPROVED** [review]

File: lib/db_utils.go (hunk 1 above)
Comment (lazynina) [reply lib/db_utils.go:11919]: I do like having the name change to `KeysOnly` tho. will just delete the old one.

**Review by tholonious — APPROVED** [review]

**Comment by lazynina** [comment]
### Merge activity

* **Mar 21, 5:05 PM EDT**: @lazynina started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/deso-protocol/core/1135).

**Pull request merged**